ProcessLib/LIE/HydroMechanics/LocalAssembler/HydroMechanicsLocalAssemblerMatrix-impl.h

```diff
@@ -353,10 +349,6 @@ computeSecondaryVariableConcreteWithBlockVectors(
             OGS_FATAL("Computation of local constitutive relation failed.");
         state = std::move(new_state);
 
-        if (!C)
-            OGS_FATAL("Computation of tangent stiffness failed.");
```

Comment:
It was a test for ptr type before. Now a valid result is always returned, unless nothing is returned at all, which is captured by `if(!solution) OGS_FATAL`.